app/pb_kits/playbook/pb_progress_step/_progress_step.scss

```diff
@@ -0,0 +1,340 @@
+@import "../tokens/colors";
+@import "../tokens/spacing";
+@import "../tokens/opacity";
+
+$pb_progress_step_circle_size: 14px;
+$pb_progress_step_connector_width: 3px;
+$pb_progress_step_connector_height_from_top: $pb_progress_step_circle_size/2 - $pb_progress_step_connector_width/2 + 2;
+
+@mixin pb_progress_step_circle($background_color, $border_color, $icon_color) {
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    align-items: center;
+    padding-right: 0;
+    overflow: hidden;
+    height: $pb_progress_step_circle_size;
+    width: $pb_progress_step_circle_size;
+    background-color: $background_color;
+    color: $icon_color;
+    border-radius: $pb_progress_step_circle_size/2;
+    border: 2px solid $border_color;
+    margin: 2px;
+}
+
+@mixin pb_progress_step_line($width, $height, $color, $margin) {
+    width: $width;
+    height: $height;
+    background-color: $color;
+    margin: $margin;
+    flex-grow: 1;
+}
+
+[class^=pb_progress_step_kit] {
+    &[class*=_horizontal] {
+        display: flex;
+        width: 100%;
+        &[class*=_icon] {
+            [class^=pb_progress_step_item_kit_complete] {
+                [class=pb_progress_step_item_wrapper] {
+                    [class=pb_progress_step_item_step_wrapper]{
+                        [class=pb_progress_step_item_step] {
+                            [class=pb_progress_step_item_step_circle] {
+                                color: $white !important;
```

Comment:
1- I tried to condense nesting of the classes, but of the nature of the nesting classes is complicated to begin with because each element needs to be granular so that styling works for both horizontal and vertical orientations. tl;dr lots of class nesting is inevitable for this kit.
2- I tried a different way to make color white, and it worked, so this part of the code is updated.